modes/language_modes.talon

```diff
@@ -1,16 +1,4 @@
-^force see sharp$: user.code_set_language_mode("csharp")
-^force see plus plus$: user.code_set_language_mode("cplusplus")
-^force go (lang|language)$: user.code_set_language_mode("go")
-^force java$: user.code_set_language_mode("java")
-^force java script$: user.code_set_language_mode("javascript")
-^force type script$: user.code_set_language_mode("typescript")
-^force markdown$: user.code_set_language_mode("markdown")
-^force python$: user.code_set_language_mode("python")
-^force terraform$: user.code_set_language_mode("terraform")
-^force are language$: user.code_set_language_mode("r")
-^force rust$: user.code_set_language_mode("rust")
-^force talon [language]$: user.code_set_language_mode("talon")
-^force scala$: user.code_set_language_mode("scala")
+^force {user.language_mode}$: user.code_set_language_mode(language_mode)
```

Comment:
That's a good point. Is there a reason we have those language-extension mappings in there at all if we don't support them? I guess so that users can context-match on them and it'll just work?

Reply:
¯\_(ツ)_/¯
These trickled in over time, I think we erred on the side of accepting 'em to help jumpstart folks.

Reply:
I think I'd be in favour of removing the unsupported ones. If somebody really wants to have a language and not upstream it they still have the option of defining the tag and override the code.language action (or just forking knausj_talon and adding it to the list).